### PR TITLE
Light/Dark styles for syntax highlighting

### PR DESCRIPTION
**Pull request by AlanDrake**
Give me feedback if this is good enough or if we should try other themes.

### CODE REVIEW

File: html/css/main.css [hunk 1]
```diff
@@ -406,4 +426,73 @@ header li {
...
+  { position: relative; left: -4em; counter-increment: source-line; }
+pre.numberSource code > span > a:first-child::before
+  { content: counter(source-line);
+    position: relative; left: -1em; text-align: right; vertical-align: baseline;
+    border: none; display: inline-block;
+    -webkit-touch-callout: none; -webkit-user-select: none;
+    -khtml-user-select: none; -moz-user-select: none;
+    -ms-user-select: none; user-select: none;
+    padding: 0 4px; width: 4em;
+    olor: #aaaaaa;
```
Comment (ericoporto) [inline html/css/main.css:459]: is this line correct?

Comment (AlanDrake) [reply html/css/main.css:459]: It was a mistake, though we don't use line numbers so I didn't notice.

**Comment by AlanDrake** [comment]
Oh well... looks like I force pushed the fix to upstream due to a bad configuration... my apologies @morganwillcock

**Comment by AlanDrake** [comment]
Well, it was technically already merged by mistake, but I'll keep this open for the discussion and then close it.
Do we want numbered lines? I guess not because it may look bad on one liners, it can be enabled by adding a class to specific code blocks but I feel it's not worth the hassle.

**Comment by morganwillcock** [comment]
Don't worry about the updating of the branch: as it is I think everything is readable. Just going by eye, I'd guess the contrast between the purple highlighting and the dark theme might be a little low, but I'm also not setup to make any kind of contrast assessment.

> Do we want numbered lines?

I'd probably say no, as the number of lines in each block is normally quite low.

**Comment by ericoporto** [comment]
@morganwillcock Is the branch good for merging? Is there something else you have in mind to tackle there?

**Comment by morganwillcock** [comment]
It is fine with me.

**Comment by ericoporto** [comment]
ok, I am working on my fork and will upstream the merge shortly

**Comment by AlanDrake** [comment]
Closing this as it's already merged and approved

**Pull request closed without merging**